docs/release/release-team-onboarding.md

```diff
@@ -55,7 +55,7 @@ Now, let's dive into the specific onboarding notes for each sub-team below.
 
 - Start by gaining a general understanding of GitHub labels and how to find issues and pull requests for the current milestone.
 - Familiarize yourself with Prow commands: The Cluster API project utilizes [Prow](https://docs.prow.k8s.io/docs/overview/) to manage CI automation. Issues and PRs are categorized by applying specific list of area labels, which helps in prioritization of that specific issue/PR during the release cycle or release process. Learn more about the available [labels](https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md#labels-that-apply-to-kubernetes-sigscluster-api-for-both-issues-and-prs) and prow [commands](https://prow.k8s.io/command-help).
-- Take a look at [TestGrid](https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api#Summary), an interactive dashboard for visualizing the CI job results of the project in a grid format!
+- Take a look at [TestGrid](https://testgrid.k8s.io/cluster-api-core-main), an interactive dashboard for visualizing the CI job results of the project in a grid format!
```

Comment:
Should this link be to main or to the dashboard group?

Reply:
Makes sense to me

Reply:
pushed up a commit to address this comment and the one above when you have a chance to take a pass 🙂